extensions/wikia/AnalyticsEngine/AnalyticsProviderKrux.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class AnalyticsProviderKrux implements iAnalyticsProvider {
+
+	private static $siteId = 'JU3_GW1b';
```

Comment:
Can we check in how many places we've got this id? Do you think it's fine keeping it in public repo?